build/ci/variables.yml

```diff
@@ -4,6 +4,7 @@ variables:
   testFeaturesEnabledEnvironmentUrl: 'https://$(deploymentName)-featuresenabled.azurewebsites.net/'
   testApplicationScope: 'https://$(deploymentName).resoluteopensource.onmicrosoft.com/.default'
   testApplicationResource: 'https://$(deploymentName).resoluteopensource.onmicrosoft.com'
+  azureStorageAccountName: 'dcmcipermanpmlxszw4sayty'
```

Comment:
Why is the account name hardcoded here?

Reply:
Here is a snippet of the pipeline resolving it now:
<img width="611" alt="image" src="https://user-images.githubusercontent.com/2955179/171288330-832ef8af-e31d-4e15-bfcd-5828492f158a.png">